### PR TITLE
[Ahuna] Delete button in menu in assistant detail view

### PR DESCRIPTION
**Pull request by philipperolet**
## Description
It's the delete button here
image

### CODE REVIEW

File: front/components/assistant/conversation/AssistantEditionMenu.tsx [hunk 1]
```diff
@@ -110,44 +122,67 @@ export function AssistantEditionMenu({
...
   })();
   return (
-    <DropdownMenu className="text-element-700">
-      <DropdownMenu.Button>{dropdownButton}</DropdownMenu.Button>
-      <DropdownMenu.Items width={220}>
-        {isAgentPublished && <DropdownMenu.SectionHeader label="Edition" />}
-        {/* Should use the router to have a better navigation experience */}
-        {isBuilder(owner) && (
+    <>
+      {" "}
```
Comment (flvndvd) [inline front/components/assistant/conversation/AssistantEditionMenu.tsx:126]: ✂️

File: front/pages/w/[wId]/assistant/assistants.tsx [hunk 2]
```diff
@@ -181,6 +181,7 @@ export default function PersonalAssistants({
           show={showDetails !== null}
           onClose={() => {
             setShowDetails(null);
+            void mutateAgentConfigurations();
```
Comment (flvndvd) [inline front/pages/w/[wId]/assistant/assistants.tsx:184]: Can we gate this change behind a `shouldMutate` and only call this when the agent configuration has actually been mutated?

File: front/components/assistant/AssistantDetails.tsx [hunk 3]
```diff
@@ -35,7 +35,11 @@ import { timeAgoFrom } from "@app/lib/utils";
 type AssistantDetailsProps = {
   owner: WorkspaceType;
   show: boolean;
-  onClose: () => void;
+  onClose: ({
+    shouldMutateAgentConfigurations,
+  }: {
+    shouldMutateAgentConfigurations: boolean;
+  }) => void | (() => void);
```
Comment (flvndvd) [inline front/components/assistant/AssistantDetails.tsx:42]:
```suggestion
  onClose: ({
    shouldMutateAgentConfigurations,
  }: {
    shouldMutateAgentConfigurations? : boolean;
  } = {}) => void);
```

**Review by flvndvd — COMMENTED** [review]
LGTM

File: front/components/assistant/AssistantDetails.tsx (hunk 3 above)
Comment (philipperolet) [reply front/components/assistant/AssistantDetails.tsx:42]: apparently can't be done on the type declaration

**Pull request merged**